vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder.go

```diff
@@ -239,6 +239,9 @@ func (feeder *clusterStateFeeder) InitFromHistoryProvider(historyProvider histor
 				PodID:         podID,
 				ContainerName: containerName,
 			}
+			if err = feeder.clusterState.AddOrUpdateContainer(containerID, nil); err != nil {
```

Comment:
It is passing nil here... It means no info regarding resources of a container. Is it ok?

Reply:
In my understanding, the AddOrUpdateContainer will be called in the LoadPods functions in RunOnce. The LoadPods will put the right request to  ContainerState. See this line please 

> 		// Container aleady exists. Possibly update the request.
>		container.Request = request

Reply:
There is no container request info until the LoadPods called in the RunOnce. So no info regarding resources of a container in the InitFromHistoryProvider function.